README.md

```diff
@@ -114,6 +114,28 @@ async function showPerson(person) {
     console.log(`${property}`);
 })(ruben);
 
+```
+### Setting the language default
+```javascript
+(async subject => {
+  const englishSubject = subject.withLangPrefs('en');
+  const labelEnglish = await englishSubject['org:memberOf'].label
```

Comment:
```suggestion
  const subjectEn = subject.withLangPrefs('en');
  const labelEn = await subjectEn['org:memberOf'].label
```

(and below as well; I think "English subject" does not work as well as "subject (in) English")